bitbucket/resource_deploy_key.go

```diff
@@ -0,0 +1,192 @@
+package bitbucket
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"regexp"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+type DeployKey struct {
+	Pk    int    `json:"pk,omitempty"`
+	Label string `json:"label,omitempty"`
+	Key   string `json:"key,omitempty"`
+}
+
+func resourceDeployKey() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDeployKeyCreate,
+		Read:   resourceDeployKeyRead,
+		Update: resourceDeployKeyUpdate,
+		Delete: resourceDeployKeyDelete,
+		Exists: resourceDeployKeyExists,
+
+		Schema: map[string]*schema.Schema{
+			"owner": &schema.Schema{
```

Comment:
The API documentation refers to this as `accountname`which is what I used in my own PR  https://github.com/terraform-providers/terraform-provider-bitbucket/pull/6/files#diff-212cd6fa62f60904d8a923069edaa6c9R43

Not sure which is better.

Reply:
Yeah I used `owner` to align with the existing resources in this repo. Atlassian's docs seem to interchangeably use `accountname` or `username` (maybe a 1.0 vs 2.0 difference), so I guess `owner` is as good as anything.